shot-android/src/androidTest/java/com/karumi/shot/compose/ScreenshotSaverTest.kt

```diff
@@ -51,6 +52,7 @@ class ScreenshotSaverTest {
         clearSdCardFiles()
     }
 
+    @SdkSuppress(maxSdkVersion = 28)
```

Comment:
Since ScreenshotSaver now just does nothing on API 29+, I couldn't think of any good tests to add; just supressing the current ones.

Reply:
It's ok :)